shared/Makefile

```diff
@@ -8,4 +8,10 @@ PROTO_DIR := ./protos/.eobjs
 CXXFLAGS += -fPIC -I../CompilerSource -I$(PROTO_DIR)
 LDFLAGS += -shared -lpng -lyaml-cpp -L../ -lProtocols -lprotobuf
 
+override CXXFLAGS += -std=c++17
+override CFLAGS += -std=c++17
+ifeq ($(UNIX_BASED), true)
+	WEBMREPOS := $(shell cd libwebm;cmake .;make;cd ..)
```

Comment:
I don't think it should be our make file's job to invoke cmake of a random library.

Reply:
You say what you don't want, without saying what you'd prefer, removing any substance from you critique that could've been there. If this can't go here, where do you suggest I put it then? I hope you know this is what knit picking is called, and we've been over it a lot. It wastes time and puts off any kind of real progress, stretching what could be done in a few days over the span of months.